src/components/GameItem.js

```diff
@@ -4,22 +4,41 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false,
+    };
+  }
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  onGameItemClick = () => {
+    console.log('Item clicked!');
+    if (!this.state.clicked && this.props.item === 'litter') this.props.incrementScoreFunc();
+    this.setState({
```

Comment:
In general, best practice in JavaScript is to avoid single-line if statements like like 21.